react-search-component/my-app/src/SearchableList.tsx

```diff
@@ -0,0 +1,39 @@
+import { Dispatch, SetStateAction, useState } from 'react';
+
+type Props = {
+  list: string[];
+  value: string;
+  setValue: Dispatch<SetStateAction<string>>;
+};
+
+export function SearchableList({ list }: { list: string[] }) {
+  const [value, setValue] = useState('');
+
+  return (
+    <>
+      <SearchBar list={list} value={value} setValue={setValue} />
+      <MakeList list={list} value={value} setValue={setValue} />
+    </>
+  );
+}
+
+function SearchBar({ value, setValue }: Props) {
```

Comment:
I suggest you stick to the React convention of using `on` as a prefix for your event handler props.